src/ophys_etl/modules/suite2p_registration/schemas.py

```diff
@@ -74,6 +76,11 @@ class Suite2PRegistrationInputSchema(argschema.ArgSchema):
                      "allows for low-frequency drift. Default value of 0.05 "
                      "is typically clipping outliers to 512 * 0.05 = 25 "
                      "pixels above or below the median trend."))
+    clip_negative = argschema.fields.Boolean(
+        required=False,
+        default=False,
+        allow_none=False,
+        description=("Whether or not to clip negative pixel values in output"))
```

Comment:
At first, when we started discussing "negative pixel values", I thought that these were something more complicated than literally "values in a frame of the video less than 0". I don't usually expect to see "negative" and "pixel" next to each other. Sorry, but could you please spell out that these are literally "values less than 0" or give a short background as to why there are negative values, and why we would want to clip them or something to that effect?